docs/Reference/API-Methods.md

```diff
@@ -2970,6 +2970,53 @@ If the boolean value is `true`, the proposal is to add a signer. If `false`, the
     The `DEBUG` API methods are not enabled by default for JSON-RPC. Use the [`--rpc-http-api`](CLI/CLI-Syntax.md#rpc-http-api) 
     or [`--rpc-ws-api`](CLI/CLI-Syntax.md#rpc-ws-api) options to enable the `DEBUG` API methods.
 
+### debug_accountRange
+
+Returns the accounts for a specified block.
+
+**Parameters**
+
+`blockHashOrNumber` : `data` - Block hash or number
+
+`txIndex` : `integer` - Transaction index from which to start
+
+`address` : `data` - Address hash from which to start
+
+`limit` : `integer` - Maximum number of account entries to return 
+
+**Returns**
+
+`result`:`object` - Account details:
+
+* `addressMap`:`object` - List of address hashes and values
```

Comment:
I think _values_ could be more specific.  They're account addresses. ie, when I used debug_accountRange, the following was returned: 
```
{
    "jsonrpc": "2.0",
    "id": 1,
    "result": {
        "addressMap": {
            "0x9467ac1c303063830309571ef53f2a569fa4866d5e97169c251e02c874157f4b": "0x9811ebc35d7b06b3fa8dc5809a1f9c52751e1deb",
            "0xb4b2d915b2a28d6500e1821ea1db87398782c82db89b3dc2d1fc2ad70589c3c5": "0x372a70ace72b02cc7f1757183f98c620254f9c8d",
            "0xbcdea13f0e9ff4fa520c658f547404b4762de6e62f6475f0eb4d1568015570ff": "0xfe3b557e8fb62b89f4916b721be55ceb828dbd73",
            "0xbf60c2629889a3ec68a7dc7524b9d03ea390a7ab9366925bcf6d21f892d3a409": "0xd6d2369a608efa48fe78ed856f0af424b55082dd"
        },
        "nextKey": "0x0000000000000000000000000000000000000000000000000000000000000000"
    }
}
```

Reply:
Thanks, will change this.